src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.ts

```diff
@@ -149,6 +159,18 @@ export class HealthComponent implements OnInit, OnDestroy {
       data.df.stats.total_bytes
     );
 
+    if (percentUsed >= 90) {
```

Comment:
The nearfull and full ratios are not fixed thresholds. So you'd need first to read the `mon_osd_nearfull_ratio` and `mon_osd_full_ratio` settings from the back-end (you can check how this is done in the "cluster > configuration" page).

However, those settings are per-osd, not per cluster, so:
* For the OSDs, let's simply trust the osdmap or the health status: in those the specific OSDs will be reported as nearfull or full, so no need to do any math.
* For the cluster, we're actually making up the concept of a cluster being near full or full, but we may use the same thresholds, so in this very case you would need to read those 2 thresholds.

Reply:
Done.